docs/GuideTVOS.md

```diff
@@ -0,0 +1,89 @@
+---
+id: guide-tvos
+title: Apple TV changes
+layout: docs
+category: Guides (Apple TV)
+permalink: docs/guide-tvos.html
+banner: ejected
+next: native-modules-android
+previous: communication-ios
+---
+
+Apple TV support has been implemented with the intention of making existing React Native iOS applications "just work" on tvOS, with few or no changes needed in the Javascript code for the applications.
```

Comment:
Use JavaScript (uppercase J, S) whenever possible, for consistency with other docs.